client/state/data-layer/wpcom/read/following/mine/index.js

```diff
@@ -19,6 +19,19 @@ import { errorNotice } from 'state/notices/actions';
 const ITEMS_PER_PAGE = 200;
 const MAX_ITEMS = 2000;
 
+export function toValidId( val ) {
```

Comment:
> So then the data-layer, which is part of the redux stuff, which is part of build, now requires all that junk too.

while we havent moved to webpack2 yet, it should fix that issue right?
Either way, I'd also be happy with a util folder